.github/workflows/gradle.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Java CI with Gradle
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: Build on Java ${{ matrix.java_version }} and OS ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        java_version: ['8', '11']
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Gradle Wrapper Validation
+      uses: gradle/wrapper-validation-action@v1
```

Comment:
Since there seem to be multiple wrappers in the repo, I am unclear if this is validating the one we actually use in `dev/transformer`.

Reply:
It does:
Run gradle/wrapper-validation-action@v1
✓ Found known Gradle Wrapper JAR files:
  3dc39ad650d40f6c029bd8ff605c6d95865d657dbfdeacdb079db0ddfffedf9f dev/gradle/wrapper/gradle-wrapper.jar
  28b330c20a9a73881dfe9702df78d4d78bf72368e8906c70080ab6932462fe9e dev/transformer/gradle/wrapper/gradle-wrapper.jar